src/utils/intervalUpdates.ts

```diff
@@ -0,0 +1,39 @@
+import { LendBorrowDayData } from '../../generated/schema'
+import { Notional, LendBorrowTrade } from '../../generated/Notional/Notional';
+import { BigInt } from '@graphprotocol/graph-ts'
+import { getMarketIndex } from '../common';
+import { convertAssetToUnderlying } from '../accounts';
+
+export function updateLendBorrowDayData(event: LendBorrowTrade): LendBorrowDayData
+{
+    let notional = Notional.bind(event.address);
+    let timestamp = event.block.timestamp.toI32()
+    let dayId = timestamp / 86400 // rounded
+    let dayStartTimestamp = dayId * 86400
+    let currencyId = event.params.currencyId as i32;
+    let maturity = event.params.maturity;
+    let marketIndex = getMarketIndex(maturity, event.block.timestamp)
+    let key = dayId.toString() + ':' + currencyId.toString() + ':' + marketIndex.toString()

```

Comment:
If you choose to have a non-flattened object (separate entities for Lend / Borrow) you will end up overwriting each other since the keys will overlap here.

Reply:
Yes, you're right. I could add another field to the key, something like 0 for borrow and 1 for lend ?

Reply:
You can also just use the text "Lend" or "Borrow", it will be more readable than 0 / 1.